CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+the rights to use your contribution. For details, visit [https://cla.opensource.microsoft.com](cla.opensource.microsoft.com).
+
+When you submit a pull request, a CLA bot will automatically determine whether you need to provide
+a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
+provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
+For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
+contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+## What is this repo for?
+
+This repo is aimed to build up a collection of resources useful to folks in the academic space who want content around Microsoft technologies, especially developing using the Cloud, in this case Azure. This includes educators wanting content to help build curriculum, students wanting further resources to expand their knowledge, hackathon organizers and attendees who want workshops, and computer society organizers who want content for events.
+
+Although most of this content is aimed at the higher education space, there is content for K-12 educators and students.
+
+## What kind of content are you after?
+
+Anything that is either platform agnostic (such as programming language content), or focuses on Azure. As long as the content is applicable to the academic audiences described above, please add it.
+
+Although the repo is in English, the content it links to is in multiple languages. We'd love as much international content as possible.
+
+## Where will this content live?
+
+Currently all the content lives in external repositories, this repo just contains a curated list of links. We will be looking to move some content 'in house' and bring it into this repo in the future.
+
+## How can I add links?
+
+Raise a PR! If you have links you would like to add, please raise a PR adding them to the relevant section.
+
+The content is divided into a number of sections:
+
+* [Learning content](./docs/content/learning-content.md) - add content here that is designed for learning, such as curriculum, content for educators, or for student groups to learn from
+* [Hackathon content](h./docs/content/hackathon-content.md) - add content here designed for student hackathons, such as workshops and templates to help get started quickly.
+* [Event content](./docs/content/event-content.md) - add content here designed for events such as on-campus meetups or computer society events
+* [K-12](./docs/content/k-12-content.md) - add content here designed for students at K-12 level (ages 4-18), or folks completely new to coding
+
+When you add a link please add the following:
+
+* Emojis to indicate the topic and language
+  
+  The following key is in use:
+  
+  * 🇬🇧 - English Language content
```

Comment:
A US flag wouldn't be loved either - I've seen many complaints in the past where language choices have English for US English, and English (UK) for English.

I would say lets go with this for now, and once in we can revamp the structure - this is something that is planned anyways as we add more content and find a good way to label it so that it can be searched and organized.